lego/components.py

```diff
@@ -122,9 +118,9 @@ def run_command(self, command: str) -> str:
             The output of the command.
         """
 
-        return self.rpyc.modules["subprocess"].check_output(command.split())
+        return self.connection.modules["subprocess"].check_output(command.split())
 
-    def get_ip(self) -> str:
+    def get_ip(self) -> ipaddress.IPv4Address:
         """Gets IP for default route interface of the remote machine."""
 
         r_socket = self.get_remote_socket(socket.AF_INET, socket.SOCK_DGRAM)
```

Comment:
Redundant code

Reply:
Doesn't see where do you think there is redundant code here, if you could elaborate more it will help me to fix/remove the redundant code.